docs/backstage-guide-v1.md

```diff
@@ -0,0 +1,313 @@
+
+### Plugin Support:
+
+Backstage’s Plugin Support allows for extensive customization, with a wide range of open-source plugins available for integration. Additionally, teams can develop custom plugins to extend the platform's functionality, tailoring it to their specific needs.
+
+## Setting Up the Development Environment
+
+Efficient development setups are crucial to maximizing productivity. In this guide, we'll simplify your workflow using a pre-configured repository with a Backstage starter app, ready for you to clone and build on.
+
+To optimize the development environment, we'll also utilize Daytona to ensure a smooth and streamlined setup.
+
+Before setting up your environment for Backstage, let's ensure you have the tools and a basic understanding to follow along smoothly. Here are the prerequisites you'll need:
+
+### Prerequisites
+
+- Preinstalled Daytona
+- Docker installed
+- Knowledge of React and Nodejs applications
+
+### Setting up Daytona for Development Workspace
+
+Setting up a local development environment often involves challenges like version conflicts and configuration headaches. [Daytona](https://www.daytona.io/) automates the setup of a Node.js environment, handles port forwarding smoothly, and executes essential post-setup commands, delivering a consistent, ready-to-use development environment with minimal manual effort.
+
+For detailed instructions on getting Daytona installed and configured, visit the [Daytona installation documentation](https://www.daytona.io/docs/installation/installation/). This guide walks you through the process, ensuring your development environment is optimized and hassle-free.
+
+### Cloning and Setting Up Your Environment for Backstage
+
+1. **Start Daytona Server:**
+* Open your terminal and start Daytona server using following command:
+```bash
+daytona serve
+```
+2. **Create Daytona Workspace:**
+```bash
+daytona create https://github.com/daytonaio-experiments/starter-backstage.git -t local
+```
+
+3. **Select Preferred IDE:**
+   
+```bash
+daytona ide
+```
+
+4. **Open the Workspace:**
+   
+```bash
+daytona code
+```
+
+After following these commands, the Backstage repository will open in your preferred IDE.
+
+### Setting Up SSO for Backstage:
+
+To enable Single Sign-On (SSO) for your Backstage application, you need administrative access to your GitHub organization. If you don't have an organization, you can create one by navigating to [GitHub Organizations](https://github.com/account/organizations).
+
+Once you have an organization, proceed to create a new OAuth App:
+
+1. Navigate to [GitHub OAuth Apps](https://github.com/settings/applications/new) and create a new OAuth App for your GitHub user.
+2. For organization-wide integration, go to `https://github.com/organizations/<Organization_Name>/settings/applications`.
```

Comment:
seems that this is not optional

Reply:
This will be necessary if we want members of our github organization to collab with us using backstage.